func/python/dict_state.py

```diff
@@ -1,9 +1,9 @@
 import pickle
 from pyfaasm.core import (
-    get_state,
-    get_state_size,
-    set_state,
-    chain_this_with_input,
+    read_state,
+    read_state_size,
+    write_state,
+    chain,
```

Comment:
Quite a few Python functions have been updated to fit with the new pyfaasm interface